lib/util/git/index.ts

```diff
@@ -1241,21 +1241,24 @@ export async function pushCommit({
   sourceRef,
   targetRef,
   files,
+  pushOptions,
 }: PushFilesConfig): Promise<boolean> {
   await syncGit();
   logger.debug(`Pushing refSpec ${sourceRef}:${targetRef ?? sourceRef}`);
   let result = false;
   try {
-    const pushOptions: TaskOptions = {
-      '--force-with-lease': null,
-      '-u': null,
-    };
+    const extraArgs = ['--force-with-lease', '-u'];
```

Comment:
seems ok, would prefer to have a separate PR, but changes are really small, so 🤷‍♂️